accelerator/core/response/ta_fetch_txn_with_uuid.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef RESPONSE_TA_FETCH_TXN_WITH_UUID_H_
+#define RESPONSE_TA_FETCH_TXN_WITH_UUID_H_
+
+#include "common/model/transaction.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file accelerator/core/response/ta_fetch_txn_with_uuid.h
+ */
+typedef enum { UNSENT, NOT_EXIST, SENT } txn_with_uuis_status_t;
```

Comment:
Is this supposed to be `txn_with_uuid_status_t`?

Reply:
I don't get it

Reply:
`uuis` -> `uuid`.

Reply:
got it!